pkg/persistence/rethinkdb/store.go

```diff
@@ -0,0 +1,99 @@
+package rethinkdb
+
+import (
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/harbor-scanner-trivy/pkg/etc"
+	"github.com/aquasecurity/harbor-scanner-trivy/pkg/harbor"
+	"github.com/aquasecurity/harbor-scanner-trivy/pkg/job"
+	"github.com/aquasecurity/harbor-scanner-trivy/pkg/persistence"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/xerrors"
+	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
+)
+
+type rethinkdbScanJob struct {
```

Comment:
You can annotate rethink fields like `rethinkdb:"id,omitempty"` so you know what the field names will look like.

Reply:
The `go-rethinkdb` library has a setting to treat `json` annotations like they are `rethinkdb` annotations. You can check in `pkg/etc/rethinkdb.go` line 30 - `r.SetTags("rethinkdb", "json")`. I've decided to do it that way so that I don't change the internal structures of `harbor-scanner-trivy`.